### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 27

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#353] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152325943-eb720f2f-8ffc-4159-b0c9-022ba8423c46.png)

**Comment by Elia-Belli** [comment]
Se pongo $p = 1 \land q = 0 \land r = 0$ la formula $CNF \in SAT$
Applicando la regola di Risoluzione avremmo proceduto all'infinito
